src/components/AppHeader.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./AppHeader.module.css";
+
+export const AppHeader = ({cardsAmount = 0 as number}) => (
```

Comment:
**nitpick:** We should not cast types this way, as this value will always be converted to number type. Pass interface instead to type it properly, eg:

```ts
interface AppHeaderProps {
cardsAmount: number;
}

export const AppHeader = ({cardsAmount}: AppHeaderProps)) {...}
```